core-web/libs/dotcms-scss/angular/dotcms-theme/components/_table.scss

```diff
@@ -1,32 +1,285 @@
 @use "variables" as *;
-p-table {
+
+// dotTable class is a wrapper for PrimeNG DataTable to apply the new
+// design system. When all tables get migrated, this class will need to be removed.
+// The legacy styles live in _datatable.scss and those styles will be removed
+
+.dotTable {
     &.p-datatable {
-        border: 1px solid $color-palette-gray-400;
         border-radius: $border-radius-md;
 
         > .p-datatable-wrapper {
             border-radius: $border-radius-md;
+            border: 1px solid $color-palette-gray-400;
         }
 
-        .p-datatable-thead {
-            > tr > th {
-                background-color: $color-palette-gray-200;
-                color: $black;
+        .p-paginator-top,
+        .p-paginator-bottom {
+            border-width: 0 0 1px 0;
+            border-radius: 0;
+        }
+
+        .p-datatable-header,
+        .p-datatable-footer {
+            border: none;
+            padding: $spacing-3;
+        }
+
+        .p-datatable-thead > tr > th {
+            background-color: $color-palette-gray-200;
+            border: 1px solid $color-palette-gray-300;
+            border-width: 0 0 1px 0;
+            color: $black;
+            font-size: $font-size-xsm;
+            font-weight: $font-weight-semi-bold;
+            padding: $spacing-1;
+            text-align: left;
+            transition: none;
+            text-wrap: nowrap;
+
+            &:first-child {
+                padding-left: $spacing-3;
+            }
+
+            &:last-child {
+                padding-right: $spacing-3;
+            }
+        }
+
+        .p-datatable-tfoot > tr > td {
+            text-align: left;
+            padding: $spacing-2;
+            border: 1px solid $color-palette-gray-300;
+            border-width: 0 0 1px 0;
+            font-weight: $font-weight-semi-bold;
+            color: $black;
+            background: $white;
+        }
+
+        .p-sortable-column {
+            outline: 0 none;
+
+            .p-sortable-column-icon {
+                display: inline-block;
+                color: $color-palette-gray-700;
+                margin-left: $spacing-1;
+            }
+
+            .p-sortable-column-badge {
+                border-radius: 50%;
+                height: 1.143rem;
+                min-width: 1.143rem;
+                line-height: 1.143rem;
+                color: $color-palette-primary;
+                background: $bg-highlight;
+                margin-left: $spacing-1;
+            }
+
+            &:not(.p-highlight):hover {
```

Comment:
When the row is selected (.highlight) dont have any hover state.
Ask to Melissa ab out it.

<img width="811" alt="Screenshot 2024-04-29 at 10 08 02 AM" src="https://github.com/dotCMS/core/assets/1909643/239b153e-b651-4f55-921c-6e4057acc2a1">

Reply:
I notice the first column (the checkboxes) has an inline width of 5%, but the Figma has a margin of 16px (1rem). In my monitor (32") the table looks bad with that 5%.
<img width="1173" alt="Screenshot 2024-04-29 at 10 16 18 AM" src="https://github.com/dotCMS/core/assets/1909643/2d94393e-b19d-4c0d-8908-c8b57bfb7bac">

<img width="349" alt="Screenshot 2024-04-29 at 10 16 57 AM" src="https://github.com/dotCMS/core/assets/1909643/18c299ca-1766-46db-99b6-934d16d8d94c">

Figma
<img width="129" alt="Screenshot 2024-04-29 at 10 12 22 AM" src="https://github.com/dotCMS/core/assets/1909643/e95dff2b-25a6-47e2-8cd4-4e68422b2b8c">

Reply:
Happend the same of the checkout, with the last column.
<img width="240" alt="Screenshot 2024-04-29 at 10 21 21 AM" src="https://github.com/dotCMS/core/assets/1909643/7ad0cb1a-b4c7-4187-8f4a-9e0485a5710b">

<img width="221" alt="Screenshot 2024-04-29 at 10 22 34 AM" src="https://github.com/dotCMS/core/assets/1909643/a26a17ce-6bd4-4424-bcc9-0e1a417377de">

Reply:
The spacing between the checkbox and the other cells, will need to be checked scenario by scenario, same with the last column menu.  the main idea is to focus in the spacing ( padding / margin ) between cells no matter the content. however I will match the storybook as reference. 

The extra pixel (41) in the header is the 1px border. 

The row height is the expected one, since the tag is making the row taller, in the design that margin is not respected.

Reply:
I think we should have a class or a selector that when the cell have a checkbox it auto set the width to it.

you can do something like: `td:has(.checkbox) {width: 40px}` or something like that.

Reply:
done.